src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
+
+import core.basesyntax.enums.Colors;
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private static final int RADIUS_OF_DEFAULT_FIGURE = 10;
+    private static final Colors COLOR_OF_DEFAULT_FIGURE = Colors.WHITE;
+    private static final int NUM_OF_FIGURE = 5;
+    private static final Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public FiguresClass getRandomFigure() {
+        int randomNumFigure = random.nextInt(NUM_OF_FIGURE);
+        if (randomNumFigure == 1) {
+            return randomCircle();
+        }
+        if (randomNumFigure == 2) {
+            return randomSquare();
+        }
+        if (randomNumFigure == 3) {
+            return randomTriangle();
+        }
+        if (randomNumFigure == 4) {
+            return randomRectangle();
+        }
+        return randomIsoscelesTrapezoid();

```

Comment:
consider making an switch-case of this one and switch through the enum of shapes for example (you can add it )  in order to make it more clear what cases do you have
`for example case CIRCLE, not case 3`